Sources/DNA/Color/Color.swift

```diff
@@ -427,43 +307,8 @@ extension CGColor {
     }
 }
 
-// MARK: - Cell UIColors
-
-@objc extension UIColor {
-    public class var defaultCellSelectedBackgroundColor: UIColor {
-        let lightSelectedColor = UIColor(r: 230, g: 235, b: 242)!
-        return dynamicColorIfAvailable(defaultColor: lightSelectedColor, darkModeColor: lightSelectedColor.withAlphaComponent(0.4))
-    }
-}
-
 extension CGColor {
     public class var defaultCellSelectedBackgroundColor: CGColor {
```

Comment:
Should this var go to `Bootstrap`?

Reply:
They aren't used in Boostrap or FinniversKit, I think we need to add the CGColor extension for only colors that we need, not for everything.